calendar_checker/calendar_checker_test.go

```diff
@@ -1,9 +1,99 @@
 package calendar_checker
 
-import "testing"
+import (
+	"google.golang.org/api/calendar/v3"
+	"testing"
+	"time"
+)
 
-func TestFoo(t *testing.T) {
-	if true != true {
-		t.Errorf("everyting is false!")
+func TestGetNextThreeEveningsFromAfternoon(t *testing.T) {
+
+	// fixture
+	afternoonStart, _ := time.Parse(time.RFC3339, "2018-02-15T15:04:05+01:00")
+
+	firstSlot := calendar.TimePeriod{
+		Start: "2018-02-15T16:04:05+01:00",
+		End:   "2018-02-15T17:04:05+01:00",
+	}
+
+	secondSlot := calendar.TimePeriod{
+		Start: "2018-02-16T18:04:05+01:00",
+		End:   "2018-02-16T21:04:05+01:00",
+	}
+
+	calendarExample := calendar.FreeBusyCalendar{Busy: []*calendar.TimePeriod{&firstSlot, &secondSlot}}
```

Comment:
Nit: You don't actually need the `calendar.FreeBusyCalendar` instance anywhere so it would be sufficient to only create a `[]*calendar.TimePeriod` to pass to `GetNextThreeEvenings()`.

Reply:
True, I don't need it. Nonetheless, I'd rather find out how to query more than one calendar, before working on this.